rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.adoc

```diff
@@ -318,12 +318,16 @@ Type::
 
 | `protocol`
 | `string`
-| protocol specifies whether the IngressController expects incoming connections to use plain TCP or whether the IngressController expects PROXY protocol. 
- PROXY protocol can be used with load balancers that support it to communicate the source addresses of client connections when forwarding those connections to the IngressController.  Using PROXY protocol enables the IngressController to report those source addresses instead of reporting the load balancer's address in HTTP headers and logs.  Note that enabling PROXY protocol on the IngressController will cause connections to fail if you are not using a load balancer that uses PROXY protocol to forward connections to the IngressController.  See http://www.haproxy.org/download/2.2/doc/proxy-protocol.txt for information about PROXY protocol. 
- The following values are valid for this field: 
- * The empty string. * "TCP". * "PROXY". 
+| protocol specifies whether the IngressController expects incoming connections to use plain TCP or whether the IngressController expects PROXY protocol.
+ PROXY protocol can be used with load balancers that support it to communicate the source addresses of client connections when forwarding those connections to the IngressController.  Using PROXY protocol enables the IngressController to report those source addresses instead of reporting the load balancer's address in HTTP headers and logs.  Note that enabling PROXY protocol on the IngressController will cause connections to fail if you are not using a load balancer that uses PROXY protocol to forward connections to the IngressController.  See http://www.haproxy.org/download/2.2/doc/proxy-protocol.txt for information about PROXY protocol.
+ The following values are valid for this field:
+ * The empty string. * "TCP". * "PROXY".
  The empty string specifies the default, which is TCP without PROXY protocol.  Note that the default is subject to change.
 
+ | `hostNetwork`
+ | `integer`
+ | hostNetwork is used to set binding ports for the Ingress Controller so that multiple Ingress Controllers can be deployed on the same node. The following node ports can be used; `httpPort: 8080`, `httpsPort: 8443`, and `statsPort: 8936`     
+
```

Comment:
The httpPort, httpsPort and statsPort are 3 new fields of the hostNetwork object where protocol already lives. All 3 are int32 type.

It might be interesting to remind the users that these 3 fields are optional, and that their default value is 80, 443, and 1936 resp. 
I'm not sure whether  it helps to put a small reminder here what we achieve when defining these ports to something other than the default.

Reply:
@sherine-k ine I'm not sure whether it helps to put a small reminder here what we achieve when defining these ports to something other than the default. >> I think we can add a note to that effect. 

My understanding is that a cluster admin can deploy multiple Ingress controllers on the same node with the node ports by setting values but what happens if default values are used I am not sure  , please elaborate

Reply:
You're right on the `small reminder here what we achieve when defining these ports`: It's there. :+1: 

Reading this a second time,   `httpPort`, `httpsPort` and `statsPort` are at the same level as `protocol`. `hostNetwork` itself is not integer, it's the structure containing `protocol`, `httpPort`, ...

What I meant was that it might be more clear, when using ports 8080, 8443, 8936, to say _for example_, so that it's clear these are not the only values possible, and especially that these are not the defaults